src/imitation/util/networks.py

```diff
@@ -107,7 +107,11 @@ def update_stats(self, batch: th.Tensor) -> None:
     def forward(self, x: th.Tensor) -> th.Tensor:
         """Updates statistics if in training mode. Returns normalized `x`."""
         if self.training:
-            self.update_stats(x)
+            # Buffered tensors' gradients cannot be cleared by `optimizer.zero_grad()`.
```

Comment:
I don't think this comment reflects our current understanding after our call, so I updated it; please edit my change as needed.

Reply:
Thanks!